components/passport/index.tsx

```diff
@@ -67,25 +67,25 @@ export function Passport({
           "0"
         )}${String(data.passportNumber).padStart(6, "0")}${
           (CURRENT_PASSPORT_VERSION + data.passportNumber) % 10
-        }HAK${String(data.dateOfBirth.getUTCFullYear()).padStart(4, "0")}${String(
-          data.dateOfBirth.getUTCMonth() + 1
-        ).padStart(2, "0")}${String(data.dateOfBirth.getUTCDate()).padStart(
+        }HAK${String(data.dateOfBirth.getFullYear()).padStart(4, "0")}${String(
+          data.dateOfBirth.getMonth() + 1
+        ).padStart(2, "0")}${String(data.dateOfBirth.getDate()).padStart(
```

Comment:
This accidentally reverts your UTCDate PR :P

Reply:
That's awesome, I love git

For real though, thanks for catching that. Slipped through as I was doing "Funky Commit Things™" to unrevert.